spec/grpc_rest_spec.rb

```diff
@@ -66,6 +66,7 @@ def test_4(req)
     it 'should be successful' do
       params = {
         test_id: 'abc',
+        some_int: 65,
```

Comment:
In this case if value is a string "65" instead, would it still work?

Reply:
Pushed the fix!